### PR TITLE
Update non-standard-users.md

### PR DESCRIPTION
**Pull request by lambasu**

### CODE REVIEW

**Comment by lambasu** [comment]
Made an update to the Anonymous users section to add link to the admin settings for Anonymous users, and remove reference to the term 'control'

**Comment by scanum** [comment]
@JohanFreelancer9 Copy editing is needed for this PR. Thanks.

File: Teams/non-standard-users.md [hunk 1]
```diff
@@ -68,6 +68,6 @@ Anonymous users can't install, update, or delete apps in meetings.
 
 ### Usage behavior and policy for anonymous users
 
-Anonymous users can't directly use apps in meetings. Native users can continue to use meetings apps if anonymous users are present. If an app sends an adaptive card in the chat, anonymous users can interact with the card.
+Anonymous users can't directly use apps in meetings. Native users can continue to use meetings apps if anonymous users are present. If an app sends an adaptive card in the chat, anonymous users can interact with the card [see https://docs.microsoft.com/en-us/microsoftteams/meeting-settings-in-teams#allow-anonymous-users-to-join-meetings].
```
Comment (JohanFreelancer9) [inline Teams/non-standard-users.md:71]:
```suggestion
Anonymous users can't directly use apps in meetings. Native users can continue to use meetings apps if anonymous users are present. If an app sends an adaptive card in the chat, anonymous users can interact with the card For more information, see [Allow anonymous users to join meetings](https://docs.microsoft.com/microsoftteams/meeting-settings-in-teams#allow-anonymous-users-to-join-meetings).
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review.

**Review by lambasu — COMMENTED** [review]
Looks good!

**Comment by scanum** [comment]
@cichur PR has been copyedited and is ready for final review, could you please check and merge? Thanks!

**Comment by JohanFreelancer9** [comment]
@lambasu - please commit my suggestion. Thanks.

**Comment by lambasu** [comment]
Committed!

**Pull request merged**